cookbooks/bcpc/recipes/zabbix-head.rb

```diff
@@ -225,3 +231,28 @@
   owner "root"
   mode "00755"
 end
+
+ruby_block "run_state_apache2_restart" do
+  block do
+    node.run_state['restart_apache2_needed'] = true
+  end
+  action :nothing
+end
+
+ruby_block "run_state_zabbix-server_restart" do
+  block do
+    node.run_state['restart_zabbix-server_needed'] = true
+  end
+  action :nothing
+end
+
+service "apache2" do
```

Comment:
I think this would overwrite the resource definition `service "apache2"` in the apache2 cookbook's recipe. Not sure if its a good idea to re-define a resource here again.